backend/01-intro-to-express/starter/src/routes/posts.js

```diff
@@ -1,7 +1,16 @@
-import express from "express";
+import express, { Router } from "express";
 import logger from "../middleware/logger.js";
-import { getPosts } from "../controllers/postsController.js";
+import { postValidator, putValidator } from "../middleware/validation.js";
+import { creatPost, deletPost, getPostByAuthor, getPostById, getPosts, updatPost } from "../controllers/postsController.js";
```

Comment:
tna7i getPostByAuthor hna tan :
```suggestion
import { creatPost, deletPost, getPostById, getPosts, updatPost } from "../controllers/postsController.js";
```